src/commands/cm/withdraw.ts

```
@@ -0,0 +1,101 @@
+            if (!candyMachineId) {
+                this.error('Candy machine ID not found');
+                return;
+            }
+
+            const candyMachinePk = publicKey(candyMachineId);
+
+            if (!flags.force) {
+                const candyMachine = await fetchCandyMachine(umi, candyMachinePk);
+                const totalItemsRemaining = Number(candyMachine.items.length) - Number(candyMachine.itemsRedeemed);
+
+                console.log(terminalColors.BgRed + terminalColors.FgWhite + 'You are about to withdraw a candy machine' + terminalColors.FgDefault + terminalColors.BgDefault);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)